server/badge/badge_test.go

```diff
@@ -70,6 +107,38 @@ func TestHandlerFeatureIsEnabled(t *testing.T) {
 	assert.NotContains(t, response, "(aa29b85)")
 }
 
+func TestHandlerFeatureProjectIsEnabled(t *testing.T) {
+	projectTests := []struct {
+		testApp     *v1alpha1.Application
+		apiEndPoint string
+		namespace   string
+		health      string
+		status      string
+		healthColor color.RGBA
+		statusColor color.RGBA
+	}{
+		{&testApp, "/api/badge?project=default", "default", "Healthy", "Synced", Green, Green},
```

Comment:
You should be creating more than one app in a project, with different statuses, and seeing the combined status for the project.